RadixWallet/Features/AccountRecoveryScan/Children/AccountRecoveryScanInProgress/AccountRecoveryScanInProgress+View.swift

```diff
@@ -119,6 +109,20 @@ public extension AccountRecoveryScanInProgress {
 				.destinations(with: store)
 			}
 		}
+
+		@ViewBuilder
+		func fixedFrameHeader(with viewStore: ViewStoreOf<AccountRecoveryScanInProgress>) -> some SwiftUI.View {
+			VStack(alignment: .center) {
+				Text(viewStore.isScanInProgress ? "Scanning for Accounts that have been included in at least on transaction, using:" : "The first \(viewStore.maxIndex) potential accounts from this signing factor were scanned.")
```

Comment:
Maybe break out the string to a variable

Reply:
Yea should be marked FIXME: Strings